apps/partner2/client/view.coffee

```diff
@@ -149,7 +150,7 @@ module.exports = class PartnerView extends Backbone.View
     criteria =
       overview: => true
       shows: => @partner.get('displayable_shows_count') > 0
-      artists: => @partner.get('partner_artists_count') > 0
+      artists: => @partner.get('partner_artists_count') > 0 and @partner.get('display_artists_section')
```

Comment:
It was, but there is now the question of displaying the artists page in grid view. https://github.com/artsy/volt/issues/2280#issuecomment-265794607 Currently, only the list view displays, no matter what setting for "Artists Layout" is selected on Volt.